packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -156,6 +156,34 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         />
       );
       break;
+    case roomNotFoundPageTrampoline():
```

Comment:
Lets be explicit about the two cases rather than putting them inside one function?